### ISSUE TITLE
ENH: Identify extracted features

### ISSUE THREAD

**Issue by jbschiratti** [mne-tools/mne-features#4]
Once features were extracted as a matrix `X, shape (n_epochs, n_features)`, it is difficult to determine which column of `X` corresponds to a given feature. It would be nice to have an optional parameter `return_as_dataframe=True` in `extract_features` to get a Pandas DataFrame like this:
```
     mean                         std            
      0        1         2         0        1        2
0 -0.01684 -0.062938 -0.108511  0.138243 -0.07428  1.009863 ...
```
This would make post-processings on the extracted features easier.

**Comment by agramfort** [comment]
In the low level function or just in the object transformer ?

**Comment by agramfort** [comment]
For extract_features yes +1 ! We just should keep low level functions with numpy arrays

**Comment by jbschiratti** [comment]
Yes, at the level of `extract_features`. However, this would require to know the output shape of each transformer in the `FeatureUnion`: not all the feature functions return arrays of shape `(n_channels,)`.

**Comment by agramfort** [comment]
What do you suggest ?

**Comment by agramfort** [comment]
what would be the columns naming scheme for the dataframe?

**Comment by jbschiratti** [comment]
Something similar to the example in the first post of the issue, I guess. 
The index of the columns would be a MultiIndex (with 2 levels). 

At level 0, we could have the alias of the feature function which was used to extract the values (for example: `mean`, `std`...) or the alias with a prefix, to avoid any confusion (for example: `feat_mean`, `feat_std`, ...). 

At the level 1, we could have `np.arange(n_channels)` for univariate features or `np.arange(n_channels * (n_channels + 1) / 2)` for bivariate features. For univariate features, this would correspond to an enumeration of the electrodes. For bivariate features, it corresponds to an enumeration of the pairs of electrodes on which the features were computed.

**Comment by agramfort** [comment]
go for hierarchical index and names 'mean', 'std'
for the level 1 index just use np.arange(n_features_of_method)
get the output shape and use np.arange

let's keep it simple and stupid.

**Comment by jbschiratti** [comment]
Issue addressed in PR #6.
